lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="wrapper-msg is-incontext urgency-low warning">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%= gettext("Oops!") %></h3>
```

Comment:
@catong Yes, this is a pattern, though errors can have a title and copy, or just copy. If you prefer only copy here, that's fine!